src/data/providers/balancer-api-provider/modules/join/types.ts

```diff
@@ -0,0 +1,14 @@
+import { Address } from "viem";
```

Comment:
You should be able to import `Address` from `/src/types` instead of viem. They are the same type, but it would be nice to keep the standard set on the rest of the code.